Sources/XcodeGenCore/Glob.swift

```diff
@@ -89,12 +89,13 @@ public class Glob: Collection {
         }
 
         let patterns = behavior.supportsGlobstar ? expandGlobstar(pattern: adjustedPattern) : [adjustedPattern]
-        
+
+        let isIncludingFiles = includeFiles
```

Comment:
This closure made `@Sendable` closure, so it's better to avoid passing mutable variables